### PR TITLE
Move all npm packages to @rails scope

### LINKED ISSUE #33083: Collect all Rails npm packages under @rails/*
We should move rails-ujs, activestorage, and actioncable under the @rails namespace, like we have @rails/webpacker.

### PR DESCRIPTION
**Pull request by javan**
Fixes #33083

```
actioncable   → @rails/actioncable
actiontext    → @rails/actiontext
activestorage → @rails/activestorage
rails-ujs     → @rails/ujs
```

TODO:
- [x] Perform the “move” by publishing all packages manually*
  - [x] https://www.npmjs.com/package/@rails/actiontext
  - [x] https://www.npmjs.com/package/@rails/actioncable
  - [x] https://www.npmjs.com/package/@rails/activestorage
  - [x] https://www.npmjs.com/package/@rails/ujs
- [x] Ensure core team members are owners of new npm packages
  - [x] `npm owner ls @rails/actiontext`
  - [x] `npm owner ls @rails/actioncable`
  - [x] `npm owner ls @rails/activestorage`
  - [x] `npm owner ls @rails/ujs`

<em>*npm doesn’t have a built-in system for moving or renaming packages</em>

### CODE REVIEW

File: RELEASING_RAILS.md [hunk 1]
```diff
@@ -113,8 +113,8 @@ then realize it is broken.
 
 IMPORTANT: The Action Cable client and Action View's UJS adapter are released
 as NPM packages, so you must have Node.js installed, have an NPM account
-(npmjs.com), and be a package owner for `actioncable` and `rails-ujs` (you can
-check this via `npm owner ls actioncable` and `npm owner ls rails-ujs`) in
+(npmjs.com), and be a package owner for `@rails/actioncable` and `@rails/ujs` (you can
```
Comment (kaspth) [inline RELEASING_RAILS.md]: As well as the packages for `activestorage` and `actiontext` now too, right?

Who can the rest of the core team as owner by the way?

**Review by kaspth — APPROVED** [review]
Cool! What happens when people next go to call `yarn install` and the packages have been renamed?

File: RELEASING_RAILS.md (hunk 1 above)
Comment (javan) [reply RELEASING_RAILS.md]: Yeah, I'll update this list. And, I'm working with Rafael to make sure the core team are all set up with publish permissions.

**Comment by javan** [comment]
> What happens when people next go to call `yarn install` and the packages have been renamed?

The packages aren't being renamed, technically. They're being published as new packages with new names. So, existing applications will need to update their `package.json`s to point to the new packages.

**Comment by connorshea** [comment]
@javan shouldn't this be mentioned in the CHANGELOG and/or Release Notes?

**Comment by javan** [comment]
@connorshea, yes thank you! Added here https://github.com/rails/rails/commit/e3d43333db2c70dac73d9b41ddd277dc8843ab32.

**Comment by benreyn** [comment]
I just ran into this issue on a project where I couldn't figure out why I wasn't getting [this fix](https://github.com/rails/rails/pull/31441) and there was no version `6.0.0` for `rails-ujs`. Does npm provide any mechanism for communicating this better? Other than digging through the changelog, issues, and source code, there was no (obvious) indication of this change. At the very least should a post-install script log a message that says something to the effect of `rails-ujs has been deprecated. Please use @rails/ujs instead` (This would involve a version bump to the old package)

**Pull request merged**